selfdrive/camerad/cameras/camera_qcom.cc

```diff
@@ -1268,7 +1282,7 @@ static void camera_open(CameraState *s, bool rear) {
           .reg_addr = 240,
           .hw_shift = 0,
           .data_type = 10,
-          .addr_type = 4,
+          .addr_type = MSM_CAMERA_I2C_ADDR_TYPE_MAX,
```

Comment:
I don't think this one is right, the MAX one shouldn't be used.

Reply:
MSM_CAMERA_I2C_ADDR_TYPE_MAX is equal 4. same with previous.

> enum msm_camera_i2c_reg_addr_type {
	MSM_CAMERA_I2C_BYTE_ADDR = 1,
	MSM_CAMERA_I2C_WORD_ADDR,
	MSM_CAMERA_I2C_3B_ADDR,
	MSM_CAMERA_I2C_ADDR_TYPE_MAX,
};

Reply:
it is 4, but I'm saying that's the wrong enum. there's probably another 4 somewhere

Reply:
i see.sorry for my bad reading.

Reply:
can't figure out where the correct enum is. so I switched them back to 4.